src/utils/Chart.js

```diff
@@ -10,8 +10,16 @@ const Chart = {
       return d[axis];
     });
 
-    const maxDigits = Math.ceil(max).toString().length - 1;
-    const minDigits = Math.floor(min).toString().length - 1;
+    //remove negative character if it exists
+    let maxString = Math.ceil(max).toString();
+    let minString = Math.floor(min).toString();
+
+    maxString = maxString.replace('-', '');
+    minString = minString.replace('-', '');
+
+    const maxDigits = maxString.length - 1;
+    const minDigits = minString.length - 1;
```

Comment:
Thinking these could all be chained.
`const maxDigits = Math.ceil(max).toString().replace('-', '').length - 1;`

Also, instead of the replace, you could do `Math.abs(Math.ceil(max)).to....` I tihink.